x-pack/packages/kbn-elastic-assistant/impl/assistant/settings/settings_context_menu/settings_context_menu.tsx

```diff
@@ -60,10 +71,15 @@ export const SettingsContextMenu: React.FC<Params> = React.memo(
       [navigateToApp]
     );
 
+    const handleNavigateToAnonymization = useCallback(() => {
+      showAnonymizationModal();
+      closePopover();
+    }, [navigateToApp]);
+
     const handleNavigateToKnowledgeBase = useCallback(
       () =>
         navigateToApp('management', {
-          path: 'kibana/securityAiAssistantManagement',
+          path: 'kibana/securityAiAssistantManagement?tab=knowledge_base',
```

Comment:
```suggestion
          path: `kibana/securityAiAssistantManagement?tab=${KNOWLEDGE_BASE_TAB}`,
```

Import this value from `const.ts`